src/Sarif.FunctionalTests/ConverterTestData/FxCop/FxCopReportAllMessageStates.xml.sarif

```diff
@@ -0,0 +1,357 @@
+{
+  "version": "1.0.0-beta.4",
+  "runs": [
+    {
+      "tool": {
+        "name": "FxCop"
+      },
+      "files": {
+        "file:///C:/Users/user/Documents/Visual Studio 2015/Projects/WpfApplication5/WpfApplication5/bin/Debug/WpfApplication5.exe": [
+          {
+            "mimeType": "application/octet-stream"
+          }
+        ],
+        "file:///c:/users/user/documents/visual studio 2015/Projects/WpfApplication5/WpfApplication5/Foo.cs": [
+          {
+            "mimeType": "text/x-csharp"
+          }
+        ]
+      },
+      "logicalLocations": {
+        "WpfApplication5": [
+          {
+            "name": "WpfApplication5",
+            "kind": "namespace"
+          }
+        ],
+        "wpfapplication5.exe": [
+          {
+            "name": "wpfapplication5.exe",
+            "kind": "module"
+          }
+        ],
+        "WpfApplication5.Foo": [
+          {
+            "name": "wpfapplication5.exe",
+            "kind": "module"
+          },
+          {
+            "name": "WpfApplication5",
+            "kind": "namespace"
+          },
+          {
+            "name": "Foo",
+            "kind": "type"
+          }
+        ],
+        "WpfApplication5.Foo.Baz()": [
+          {
+            "name": "wpfapplication5.exe",
+            "kind": "module"
+          },
+          {
+            "name": "WpfApplication5",
+            "kind": "namespace"
+          },
+          {
+            "name": "Foo",
+            "kind": "type"
+          },
+          {
+            "name": "#Baz()",
+            "kind": "member"
+          }
+        ],
+        "WpfApplication5.MainWindow.System.Windows.Markup.IComponentConnector.Connect(System.Int32,System.Object)": [
+          {
+            "name": "wpfapplication5.exe",
+            "kind": "module"
+          },
+          {
+            "name": "WpfApplication5",
+            "kind": "namespace"
+          },
+          {
+            "name": "MainWindow",
+            "kind": "type"
+          },
+          {
+            "name": "#System.Windows.Markup.IComponentConnector.Connect(System.Int32,System.Object)",
+            "kind": "member"
+          }
+        ],
+        "WpfApplication5.Properties.Resources": [
+          {
+            "name": "wpfapplication5.exe",
+            "kind": "module"
+          },
+          {
+            "name": "WpfApplication5.Properties",
+            "kind": "namespace"
+          },
+          {
+            "name": "Resources",
+            "kind": "type"
+          }
+        ],
+        "WpfApplication5.Properties.Resources..ctor()": [
+          {
+            "name": "wpfapplication5.exe",
+            "kind": "module"
+          },
+          {
+            "name": "WpfApplication5.Properties",
```

Comment:
Yes. separate issue. The FxCop converter should break these apart. There is also the open question of whether the log file should contain the root container as an item if it matches the analysis target.